travelportal.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
change this file name to passenger.jso

Reply:
Changes are done. Please review the Changes